docker-compose.test.yml

```diff
@@ -12,7 +12,7 @@ services:
 
   backend:
     # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
-    image: ghcr.io/ocelot-social-community/ocelot-social/backend:test
+    image: ocelotsocialnetwork/backend:test
```

Comment:
for testing it is ok to change images, but we need this to be set to the github image repository